pkg/services/accesscontrol/authorizer.go

```diff
@@ -128,6 +128,8 @@ func (c *LegacyAccessClient) Check(ctx context.Context, id claims.AuthInfo, req
 	} else if req.Verb == utils.VerbList {
 		// For list request we need to filter out in storage layer.
 		eval = EvalPermission(action)
+	} else if req.Verb == utils.VerbCreate || req.Verb == utils.VerbUpdate || req.Verb == utils.VerbPatch || req.Verb == utils.VerbDelete {
+		eval = EvalPermission(action)
```

Comment:
For my edification, is there a particular reason you didn't combine this with the if statement right above it?

Reply:
😅 I was using some other previously, but at the end I figured out that I don't need that and I haven't realized that that's the same as the previous branch -> I'm gonna simplify/update it!